### PR TITLE
feat: add new rule rrd - vfor with index key w tests

### PR DESCRIPTION
**Pull request by David-Pena**

### CODE REVIEW

File: docs/rules/rrd/v-for-with-index-key.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## 🤩 How to fix it?
+
+::: tip
+To ensure stable rendering and improve performance, use a unique identifier from the items themselves as the `:key`. This could be an id or another unique property.
+:::
+
+```vue
+<template>
```
Comment (rrd108) [inline docs/rules/rrd/v-for-with-index-key.md:33]: I really like using the same examples in the docs and in the tests. Somehow it just feels right and consistent.

Perhaps it is not possible to all cases.

🤩

**Review by rrd108 — APPROVED** [review]

File: docs/rules/rrd/v-for-with-index-key.md (hunk 1 above)
Comment (David-Pena) [reply docs/rules/rrd/v-for-with-index-key.md:33]: Oh I see, the example in the docs is missing the <p> tag I think.

> Perhaps it is not possible to all cases

For example if this rule's tests have 3 where it should report error, you mean adding those three to the docs?

Or just taking one of the tests to be the docs example?

Comment (rrd108) [reply docs/rules/rrd/v-for-with-index-key.md:33]: One of them will work.

Comment (David-Pena) [reply docs/rules/rrd/v-for-with-index-key.md:33]: Updated in latest commit https://github.com/rrd108/vue-mess-detector/commit/202a0c21e72cb82089088c7e6cbebd60678b4d89

**Pull request merged**